player/lua/osc.lua

```diff
@@ -1866,7 +1866,9 @@ local function osc_init()
         return ("\238\132\134" .. osc_styles.smallButtonsLlabel .. " " ..
                (mp.get_property_native("aid") or "-") .. "/" .. audio_track_count)
     end
-    ne.eventresponder["mbtn_left_up"] = command_callback(user_opts.audio_track_mbtn_left_command)
+    ne.eventresponder["mbtn_left_up"] = command_callback(audio_track_count == 1
+        and "cycle audio"
```

Comment:
The previous version checked if the default command was changed and sfan made me change it.  ¯\\_(ツ)_/¯

Reply:
That depends on what the purpose of the custom command was.
Is it so the user can use his own selection script? cycling would still make sense in that case.
Or so a totally custom action can be defined for the button? (use case?)

Either way the alternative is to change the default to `""` and use that to indicate this special behavior.

Reply:
We can also just leave it as it is now that right click cycles tracks again. We keep the UI consistent as kasper noted, and if you want to cycle 1 track you can right click. Otherwise I would rather add 2 more script-opts for commands with 1 track than a script message.

Reply:
> You just add this custom handling as a script message, then it can be written as a command.

This would be proper solution. Having hardcoded conditions/commands beats the purpose of having it customizable. We cannot assume certain use of those buttons, if they are customizable. I already mentioned that about `enabled` conditions, that currently prevent to customize forward/back buttons.